spec/report_spec.rb

```diff
@@ -0,0 +1,46 @@
+require_relative 'spec_helper'
+
+describe RepoAudit::Report do
+  let(:repo) { double(Github::Mash, { full_name: 'some-user/not-a-real-repo' }) }
+  let(:report) { described_class.new(repo: repo) }
+  let(:checker) { double(run: nil) }
+
+  before do
+    allow(RepoAudit::LicenseChecker).to receive(:new).with(repo).and_return(checker)
+    allow(ENV).to receive(:fetch).with('GH_TOKEN').and_return('dummy-token')
```

Comment:
Oh, I'm not saying it shouldn't use an env var, just that the `ENV.fetch` maybe shouldn't be in `RepoAudit::Report.fetch_repo()`. (Rather, that we should pass the GitHub token to `RepoAudit::Report.initializer()` having `ENV.fetch`'d in `bin/whatever.rb`.)